content/docs/getting_started/visual_story/validation@es.md

```diff
@@ -0,0 +1,28 @@
+---
+$title: Validar el código AMP HTML 
+$order: 7
```

Comment:
Please remove the added frontmatter from all the files. Only the translated $title should be present on translated files. Thank you.

Reply:
Adapted front matter in the translated files for validation.md.